frontend/src/concepts/pipelines/content/tables/pipelineRun/PipelineRunTableToolbar.tsx

```diff
@@ -28,9 +28,14 @@ const PipelineRunTableToolbar: React.FC<PipelineRunTableToolbarProps> = ({
 }) => {
   const { versions } = React.useContext(PipelineRunVersionsContext);
   const { experimentId, pipelineVersionId } = useParams();
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const { [RuntimeStateKF.RUNTIME_STATE_UNSPECIFIED]: unspecifiedState, ...statusRuntimeStates } =
-    runtimeStateLabels;
+  /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
this needs to be done for the compare runs add page too

Reply:
Fixed

<img width="1238" alt="Screenshot 2024-09-03 at 5 45 15 PM" src="https://github.com/user-attachments/assets/05f4efe5-dddf-4e39-987e-1c9fd7842d60">